packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -684,7 +698,7 @@ function MapAnalysisImpl(props: ImplProps) {
             {
               type: 'item',
               // concatenating the parent and subMenu labels creates a unique ID
-              id: MapSideNavItemLabels.ConfigureMap + MarkerTypeLabels.barplot,
+              id: 'single-variable-bar',
```

Comment:
nit: same as above; comment can be removed

Reply:
Yes!